.travis.yml

```diff
@@ -199,6 +199,28 @@ jobs:
       before_cache:
         - conda clean --all --yes
 
+    - language: python
```

Comment:
Is there really no way to DRY this?

Reply:
Build matrix expansion and build stages could help, but with build stages we'd lose some concurrency and things are arguably less explicit with matrix expansion. Refactoring of the travis config can happen in a follow-on PR.